app/admin/admin.module.ts

```diff
@@ -10,18 +12,22 @@ import { UsersFilter } from './pipes/filter.pipe';
 import { UsersSorter } from './pipes/sort.pipe';
 import { adminRouting } from './admin.routing';
 import { UsersListComponent } from './users-list/users-list.component';
+import { EditUserComponent } from './edit-user/edit-user.component';
 
 @NgModule({
   imports: [
     BrowserModule,
     adminRouting,
     FormsModule,
     MdCardModule,
+    MdGridListModule,
+    MdInputModule,
```

Comment:
Instead of importing Md... Modules, it's maybe better to import the MaterialModule.